### PR TITLE
Schedule reviews of MQTT and SQLite

### PR DESCRIPTION
**Pull request by mloskot**
Please, @klemens-morgenstern, if you could provide the missing GitHub links and MQTT submitter, thanks.

### CODE REVIEW

**Comment by klemens-morgenstern** [comment]
Ivica Siladic submitted https://github.com/mireo/async-mqtt5

**Comment by mloskot** [comment]
@klemens-morgenstern We need link to GitHub repo or even specific branch if non-default is supposed to be reviewed
and we also need Documentation link, for example, see https://github.com/boostorg/website/commit/6cb330902278cd09aa00ae641ef02d94356994e1

File: community/review_schedule.html [hunk 1]
```diff
@@ -72,6 +72,31 @@ <h2><a name="Schedule" id="Schedule"></a>Schedule</h2>
                   </tr>
                   -->
 
+                  <tr>
+                    <td>MQTT</td>
+                    <td></td>
+                    <td>
+                      <ul>
+                        <li><a href="">Github</a></li>
```
Comment (klemens-morgenstern) [inline community/review_schedule.html:80]:
```suggestion
                        <li><a href="https://github.com/mireo/async-mqtt5/tree/master">Github</a></li>
```

File: community/review_schedule.html [hunk 2]
```diff
@@ -72,6 +72,31 @@ <h2><a name="Schedule" id="Schedule"></a>Schedule</h2>
...
+                        <li><a href="">Documentation</a></li>
```
Comment (klemens-morgenstern) [inline community/review_schedule.html:81]:
```suggestion
                        <li><a href="https://spacetime.mireo.com/async-mqtt5/>Documentation</a></li>
```

File: community/review_schedule.html [hunk 3]
```diff
@@ -72,6 +72,31 @@ <h2><a name="Schedule" id="Schedule"></a>Schedule</h2>
...
+                    <td>October 16, 2024 - October 25, 2024</td>
+                  </tr>
+
+                  <tr>
+                    <td>SQLite</td>
+                    <td>Klemens Morgenstern</td>
+                    <td>
+                      <ul>
+                        <li><a href="">Github</a></li>
+                        <li><a href="">Documentation</a></li>
```
Comment (klemens-morgenstern) [inline community/review_schedule.html:94]:
```suggestion
                        <li><a href="https://klemens.dev/sqlite/">Documentation</a></li>
```

File: community/review_schedule.html [hunk 4]
```diff
@@ -72,6 +72,31 @@ <h2><a name="Schedule" id="Schedule"></a>Schedule</h2>
...
+                    <td>Klemens Morgenstern</td>
+                    <td>October 16, 2024 - October 25, 2024</td>
+                  </tr>
+
+                  <tr>
+                    <td>SQLite</td>
+                    <td>Klemens Morgenstern</td>
+                    <td>
+                      <ul>
+                        <li><a href="">Github</a></li>
```
Comment (klemens-morgenstern) [inline community/review_schedule.html:93]:
```suggestion
                        <li><a href="https://github.com/klemens-morgenstern/sqlite">Github</a></li>
```

**Review by klemens-morgenstern — CHANGES_REQUESTED** [review]

**Pull request merged**